### DISCUSSION TITLE
the code box caption overflow hidden

### DISCUSSION THREAD

**Discussion by aidevtao** [GitbookIO/community#193] [0. General bugs and questions]
### What's your feature/idea?

<img width="695" alt="image" src="https://user-images.githubusercontent.com/130148608/235656538-a57bd8e7-e4b2-41ec-9623-fab79d51ebda.png">

### What's your desired outcome?

<img width="695" alt="image" src="https://user-images.githubusercontent.com/130148608/235656569-8511be30-f398-429e-96f7-541a2b5bcced.png">

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

### Anything else?

![Uploading image.png…]()

**Comment by aidevtao** [comment]
why hidden the path of file name .
i cant understand
i can't say anything

**Reply by aikoii** [reply]
Hi @aidevtao,

Sorry to hear you are having those issues!

Could you email us at [support@gitbook.com](mailto:support@gitbook.com)? Please include the URL of the space you are working with in the email and pages where you see this happening.

Thank you!
